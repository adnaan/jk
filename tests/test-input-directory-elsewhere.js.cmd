jk run --input-directory . input/test.js
