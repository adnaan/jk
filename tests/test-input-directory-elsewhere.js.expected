this is from input.json in a directory given with --input-directory
